src/shared/actions/transfers.js

```diff
@@ -433,123 +434,130 @@ export const makeTransaction = (seedStore, receiveAddress, value, message, accou
     const withPreTransactionSecurityChecks = () => {
         // Progressbar step => (Validating receive address)
         dispatch(setNextStepAsActive());
+
         // Check the last trit for validity
-        if (isLastTritZero(address) === false){
-          throw new Error(Errors.INVALID_LAST_TRIT);
-        }
-        // Make sure that the address a user is about to send to is not already used.
-        return shouldAllowSendingToAddress()([address])
-            .then((shouldAllowSending) => {
-                if (shouldAllowSending) {
-                    // Progressbar step => (Syncing account)
-                    dispatch(setNextStepAsActive());
+        return isLastTritZeroAsync(address).then((lastTritIsZero) => {
```

Comment:
This is an async version of `isLastTritZero` so that it can be integrated into the promise chain and caught in the `catch` block: https://github.com/iotaledger/trinity-wallet/blob/8dbed4589d5bdf139d00a1579c31d8603b7acf4e/src/shared/actions/transfers.js#L709